contracts/lib/arbitration/IArbitrable.sol

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
It looks like this and the next contract are duplicated in `apps/disputable` (they probably belong better in `lib/` actually)